Source/Boxed.AspNetCore.TagHelpers/OpenGraph/ObjectTypes/OpenGraphMetadata.cs

```diff
@@ -359,13 +361,10 @@ protected virtual void Validate()
         private string GetRequestUrl()
         {
             var httpContext = this.ViewContext.HttpContext;
-            var urlHelper = httpContext
-                .Features
-                .Get<IServiceProvidersFeature>()
-                .RequestServices
-                .GetRequiredService<IUrlHelper>();
+            var urlHelper = httpContext.GetUrlHelper();
             var request = httpContext.Request;
-            return new Uri(new Uri(request.Scheme + "://" + request.Host.Value), urlHelper.Content(request.Path.Value)).ToString();
+            var baseUri = new Uri(string.Concat(request.Scheme, "://", request.Host.Value));
```

Comment:
Can we not do this:

```
var uriBuilder = new UriBuilder(request.Scheme, request.Host.Value)
{
    Path = urlHelper.Content(request.Path.Value),
};
return uriBuilder.ToString();
```

Reply:
I don't think so. Setting `Path` escapes string